README.md

```diff
@@ -48,6 +48,23 @@ First-timers-bot is built with [Probot](https://probot.github.io/).
 
 <p align="center"><img src="/assets/Issue-Done.png"></p>
 
+### Configuration
+
+If you would like to add your own configuration, add a file called 'first-timers.yml' to the .github directory of the repository where the app is installed with the following options. 
+
+```
+#you can create the issue in a different repo than where the problem is. Just make sure you've given the bot access to all the repos where you would like to create issues. The issue will link back to the original repository where the contribution will be made.
+repository:
+    - name: repo-name
```

Comment:
maybe just

```yml
repository: repo-name
```

Reply:
Maybe comment it out by default?

> Just make sure you've given the bot access to all the repos where you would like to create issues

Just make sure you installed the bot the the configured repository